TKeazirian.HTTPServer/Handler/FileServerHandler.cs

```diff
@@ -0,0 +1,41 @@
+namespace TKeazirian.HTTPServer.Handler;
+
+using Response;
+using Request;
+
+public class FileServerHandler : Handler
+{
+    public override Response HandleResponse(Request request)
+    {
+        string requestPath = request.GetRequestPath();
+        string fileExtension = Path.GetExtension(requestPath);
+        string fileContentType = GetFileContentType(fileExtension);
+        string body;
+        try
+        {
+            body = File.ReadAllText($@"./Resources{requestPath}");
+        }
+        catch
+        {
+            throw new FileNotFoundException("The file cannot be found.");
```

Comment:
Could other errors occur when trying to read the file? i.e. could a file exist, but not be read for some reason?

Reply:
Good call - I will make the error message more generic.  And add a log as Marie mentioned above.